CHANGELOG.md

```diff
@@ -4,6 +4,58 @@ All notable changes to this project will be documented in this file. And this pr
 
 ## [Unreleased]
 
+### Framework
+
+- [Optimization] Update default rust toolchain to `v1.34.0`. [@yangby-cryptape] [@kaikai1024]
+- [Optimization] Update cita-sdk-js version. [@kaikai1024]
+- [Fix] Reorganize toml path. [@ouwenkg]
+- [Feature] Log output mode can be configured as stdout or file. [@Kayryu]
+
+### Executor
+
+- [Optimization] Integrate vm-test. [@ouwenkg]
+- [Optimization] Add unit test of calling contract. [@kaikai1024]
+
+### Auth
+
+- [Fix] Auth crashes when it is not ready. [@leeyr338]
+
+### Network
+
+- [Fix] High CPU usage. [@leeyr338]
+- [Fix] Refuse connect when reach max connections. [@leeyr338]
+- [Optimization] Add discovery test of network. [@leeyr338]
+- [Optimization] Use new version `p2p` API. [@jerry-yu]
```

Comment:
Use new version `p2p` can fix this issue, It is better clearly written here.
https://github.com/cryptape/cita/issues/472

Reply:
ok